### ISSUE TITLE
package-info.java:29-32: Continue replacing usage of...

### ISSUE THREAD

**Issue by 0pdd** [yegor256/cactoos#1117]
The puzzle `1033-b5541bca` from #1033 has to be resolved: 

https://github.com/yegor256/cactoos/blob/b74f80cca44fa8b9bf58213159c2a068151da6c4/src/main/java/org/cactoos/io/package-info.java#L29-L32

The puzzle was created by Vytas on 21-Apr-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#1117](https://github.com/yegor256/cactoos/issues/1117) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#1117](https://github.com/yegor256/cactoos/issues/1117) assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/096b24d4-d84c-4020-b033-dfc961925c82); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by victornoel** [comment]
@0crat waiting for #1086 to be solved, without latest cactoos-matchers, it's hell to test things like `BytesOf` that have side-effect when reading from streams (can read only once).

**Comment by g4s8** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/cactoos/issues/1117#issuecomment-497327043))

@g4s8 This is what I know about this job in [C63314D6Z](https://www.0crat.com/p/C63314D6Z), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1117](https://github.com/yegor256/cactoos/issues/1117) is [in scope](http://www.zerocracy.com/policy.html#14) for 1mon
  * The role is `DEV`
  * The job is assigned to @victornoel[/z](https://www.0crat.com/u/victornoel) for [5days](http://www.zerocracy.com/policy.html#8)
  * There is no monetary reward attached, it's a [free](http://www.zerocracy.com/policy.html#2) job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @0pdd[/z](https://www.0crat.com/u/0pdd): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Ayegor256%2Fcactoos%231117%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by victornoel** [comment]
@0crat waiting for #1086 to be solved, without latest cactoos-matchers, it's hell to test things like BytesOf that have side-effect when reading from streams (can read only once).

**Comment by 0crat** [comment]
> @0crat waiting for #1086 to be solved, without latest cactoos-matchers, it's hell to test things ... ([here](https://github.com/yegor256/cactoos/issues/1117#issuecomment-497349103))

@victornoel The impediment for [#1117](https://github.com/yegor256/cactoos/issues/1117) was registered successfully by @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by victornoel** [comment]
@0crat wait for #1145 to be merged

**Comment by 0crat** [comment]
> @0crat wait for #1145 to be merged ([here](https://github.com/yegor256/cactoos/issues/1117#issuecomment-504671080))

@victornoel Job [#1117](https://github.com/yegor256/cactoos/issues/1117) is already on hold

**Comment by 0pdd** [comment]
The puzzle `1033-b5541bca` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1152](https://github.com/yegor256/cactoos/issues/1152) is still not solved.

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1117](https://github.com/yegor256/cactoos/issues/1117) is now out of scope

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1182](https://github.com/yegor256/cactoos/issues/1182) is still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1228](https://github.com/yegor256/cactoos/issues/1228) is still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1235](https://github.com/yegor256/cactoos/issues/1235) is still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1331](https://github.com/yegor256/cactoos/issues/1331) is still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1343](https://github.com/yegor256/cactoos/issues/1343), [#1360](https://github.com/yegor256/cactoos/issues/1360) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1360](https://github.com/yegor256/cactoos/issues/1360), [#1362](https://github.com/yegor256/cactoos/issues/1362) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1360](https://github.com/yegor256/cactoos/issues/1360), [#1367](https://github.com/yegor256/cactoos/issues/1367) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343), [#1362](https://github.com/yegor256/cactoos/issues/1362).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1367](https://github.com/yegor256/cactoos/issues/1367), [#1372](https://github.com/yegor256/cactoos/issues/1372) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343), [#1360](https://github.com/yegor256/cactoos/issues/1360), [#1362](https://github.com/yegor256/cactoos/issues/1362).

**Comment by 0pdd** [comment]
@0pdd 3 puzzles [#1372](https://github.com/yegor256/cactoos/issues/1372), [#1375](https://github.com/yegor256/cactoos/issues/1375), [#1376](https://github.com/yegor256/cactoos/issues/1376) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343), [#1360](https://github.com/yegor256/cactoos/issues/1360), [#1362](https://github.com/yegor256/cactoos/issues/1362), [#1367](https://github.com/yegor256/cactoos/issues/1367).

**Comment by 0pdd** [comment]
@0pdd 3 puzzles [#1375](https://github.com/yegor256/cactoos/issues/1375), [#1376](https://github.com/yegor256/cactoos/issues/1376), [#1381](https://github.com/yegor256/cactoos/issues/1381) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343), [#1360](https://github.com/yegor256/cactoos/issues/1360), [#1362](https://github.com/yegor256/cactoos/issues/1362), [#1367](https://github.com/yegor256/cactoos/issues/1367), [#1372](https://github.com/yegor256/cactoos/issues/1372).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1375](https://github.com/yegor256/cactoos/issues/1375), [#1381](https://github.com/yegor256/cactoos/issues/1381) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343), [#1360](https://github.com/yegor256/cactoos/issues/1360), [#1362](https://github.com/yegor256/cactoos/issues/1362), [#1367](https://github.com/yegor256/cactoos/issues/1367), [#1372](https://github.com/yegor256/cactoos/issues/1372), [#1376](https://github.com/yegor256/cactoos/issues/1376).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1381](https://github.com/yegor256/cactoos/issues/1381), [#1389](https://github.com/yegor256/cactoos/issues/1389) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343), [#1360](https://github.com/yegor256/cactoos/issues/1360), [#1362](https://github.com/yegor256/cactoos/issues/1362), [#1367](https://github.com/yegor256/cactoos/issues/1367), [#1372](https://github.com/yegor256/cactoos/issues/1372), [#1375](https://github.com/yegor256/cactoos/issues/1375), [#1376](https://github.com/yegor256/cactoos/issues/1376).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1381](https://github.com/yegor256/cactoos/issues/1381), [#1391](https://github.com/yegor256/cactoos/issues/1391) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343), [#1360](https://github.com/yegor256/cactoos/issues/1360), [#1362](https://github.com/yegor256/cactoos/issues/1362), [#1367](https://github.com/yegor256/cactoos/issues/1367), [#1372](https://github.com/yegor256/cactoos/issues/1372), [#1375](https://github.com/yegor256/cactoos/issues/1375), [#1376](https://github.com/yegor256/cactoos/issues/1376), [#1389](https://github.com/yegor256/cactoos/issues/1389).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1391](https://github.com/yegor256/cactoos/issues/1391), [#1392](https://github.com/yegor256/cactoos/issues/1392) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343), [#1360](https://github.com/yegor256/cactoos/issues/1360), [#1362](https://github.com/yegor256/cactoos/issues/1362), [#1367](https://github.com/yegor256/cactoos/issues/1367), [#1372](https://github.com/yegor256/cactoos/issues/1372), [#1375](https://github.com/yegor256/cactoos/issues/1375), [#1376](https://github.com/yegor256/cactoos/issues/1376), [#1381](https://github.com/yegor256/cactoos/issues/1381), [#1389](https://github.com/yegor256/cactoos/issues/1389).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1391](https://github.com/yegor256/cactoos/issues/1391), [#1395](https://github.com/yegor256/cactoos/issues/1395) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343), [#1360](https://github.com/yegor256/cactoos/issues/1360), [#1362](https://github.com/yegor256/cactoos/issues/1362), [#1367](https://github.com/yegor256/cactoos/issues/1367), [#1372](https://github.com/yegor256/cactoos/issues/1372), [#1375](https://github.com/yegor256/cactoos/issues/1375), [#1376](https://github.com/yegor256/cactoos/issues/1376), [#1381](https://github.com/yegor256/cactoos/issues/1381), [#1389](https://github.com/yegor256/cactoos/issues/1389), [#1392](https://github.com/yegor256/cactoos/issues/1392).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1395](https://github.com/yegor256/cactoos/issues/1395), [#1396](https://github.com/yegor256/cactoos/issues/1396) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343), [#1360](https://github.com/yegor256/cactoos/issues/1360), [#1362](https://github.com/yegor256/cactoos/issues/1362), [#1367](https://github.com/yegor256/cactoos/issues/1367), [#1372](https://github.com/yegor256/cactoos/issues/1372), [#1375](https://github.com/yegor256/cactoos/issues/1375), [#1376](https://github.com/yegor256/cactoos/issues/1376), [#1381](https://github.com/yegor256/cactoos/issues/1381), [#1389](https://github.com/yegor256/cactoos/issues/1389), [#1391](https://github.com/yegor256/cactoos/issues/1391), [#1392](https://github.com/yegor256/cactoos/issues/1392).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1395](https://github.com/yegor256/cactoos/issues/1395), [#1420](https://github.com/yegor256/cactoos/issues/1420) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343), [#1360](https://github.com/yegor256/cactoos/issues/1360), [#1362](https://github.com/yegor256/cactoos/issues/1362), [#1367](https://github.com/yegor256/cactoos/issues/1367), [#1372](https://github.com/yegor256/cactoos/issues/1372), [#1375](https://github.com/yegor256/cactoos/issues/1375), [#1376](https://github.com/yegor256/cactoos/issues/1376), [#1381](https://github.com/yegor256/cactoos/issues/1381), [#1389](https://github.com/yegor256/cactoos/issues/1389), [#1391](https://github.com/yegor256/cactoos/issues/1391), [#1392](https://github.com/yegor256/cactoos/issues/1392), [#1396](https://github.com/yegor256/cactoos/issues/1396).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1395](https://github.com/yegor256/cactoos/issues/1395), [#1425](https://github.com/yegor256/cactoos/issues/1425) are still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343), [#1360](https://github.com/yegor256/cactoos/issues/1360), [#1362](https://github.com/yegor256/cactoos/issues/1362), [#1367](https://github.com/yegor256/cactoos/issues/1367), [#1372](https://github.com/yegor256/cactoos/issues/1372), [#1375](https://github.com/yegor256/cactoos/issues/1375), [#1376](https://github.com/yegor256/cactoos/issues/1376), [#1381](https://github.com/yegor256/cactoos/issues/1381), [#1389](https://github.com/yegor256/cactoos/issues/1389), [#1391](https://github.com/yegor256/cactoos/issues/1391), [#1392](https://github.com/yegor256/cactoos/issues/1392), [#1396](https://github.com/yegor256/cactoos/issues/1396), [#1420](https://github.com/yegor256/cactoos/issues/1420).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1425](https://github.com/yegor256/cactoos/issues/1425) is still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343), [#1360](https://github.com/yegor256/cactoos/issues/1360), [#1362](https://github.com/yegor256/cactoos/issues/1362), [#1367](https://github.com/yegor256/cactoos/issues/1367), [#1372](https://github.com/yegor256/cactoos/issues/1372), [#1375](https://github.com/yegor256/cactoos/issues/1375), [#1376](https://github.com/yegor256/cactoos/issues/1376), [#1381](https://github.com/yegor256/cactoos/issues/1381), [#1389](https://github.com/yegor256/cactoos/issues/1389), [#1391](https://github.com/yegor256/cactoos/issues/1391), [#1392](https://github.com/yegor256/cactoos/issues/1392), [#1395](https://github.com/yegor256/cactoos/issues/1395), [#1396](https://github.com/yegor256/cactoos/issues/1396), [#1420](https://github.com/yegor256/cactoos/issues/1420).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1559](https://github.com/yegor256/cactoos/issues/1559) is still not solved; solved: [#1152](https://github.com/yegor256/cactoos/issues/1152), [#1182](https://github.com/yegor256/cactoos/issues/1182), [#1228](https://github.com/yegor256/cactoos/issues/1228), [#1235](https://github.com/yegor256/cactoos/issues/1235), [#1331](https://github.com/yegor256/cactoos/issues/1331), [#1340](https://github.com/yegor256/cactoos/issues/1340), [#1341](https://github.com/yegor256/cactoos/issues/1341), [#1343](https://github.com/yegor256/cactoos/issues/1343), [#1360](https://github.com/yegor256/cactoos/issues/1360), [#1362](https://github.com/yegor256/cactoos/issues/1362), [#1367](https://github.com/yegor256/cactoos/issues/1367), [#1372](https://github.com/yegor256/cactoos/issues/1372), [#1375](https://github.com/yegor256/cactoos/issues/1375), [#1376](https://github.com/yegor256/cactoos/issues/1376), [#1381](https://github.com/yegor256/cactoos/issues/1381), [#1389](https://github.com/yegor256/cactoos/issues/1389), [#1391](https://github.com/yegor256/cactoos/issues/1391), [#1392](https://github.com/yegor256/cactoos/issues/1392), [#1395](https://github.com/yegor256/cactoos/issues/1395), [#1396](https://github.com/yegor256/cactoos/issues/1396), [#1420](https://github.com/yegor256/cactoos/issues/1420), [#1425](https://github.com/yegor256/cactoos/issues/1425).
